sound/soc/intel/boards/sof_da7219.c

```diff
@@ -15,11 +15,13 @@
 #include <sound/soc-acpi.h>
 #include "../../codecs/da7219.h"
 #include "hda_dsp_common.h"
+#include "sof_maxim_common.h"
+
+/* Speaker amp type */
+#define SOF_MAX98360A_SPEAKER_AMP_PRESENT	BIT(0)
+#define SOF_MAX98373_SPEAKER_AMP_PRESENT	BIT(1)
```

Comment:
do you really need this bit if it can be detected with ACPI?

Reply:
I'd like to do it later because I need to backport these patches back to older branch. Would like to minimize the porting.

Reply:
probably worth adding a comment then